command/login.go

```diff
@@ -97,22 +97,21 @@ func (cmd *Login) DoRun(args []string) error {
 	}
 
 	if len(list.Projects) == 0 {
-		cmd.ui.Info("Successful login, but you don't have any project. Please contact mayday@nerdalize.com.")
+		cmd.ui.Info("Successful login, but you don't have any cluster. Please contact mayday@nerdalize.com.")
 		return nil
 	}
 	var projectSlug string
 	for _, project := range list.Projects {
 		projectSlug = project.Nk
 		err = setProject(cmd.opts.KubeConfig, cmd.opts.Config, project, cmd.outputter.Logger)
 		if err != nil {
-			cmd.ui.Info(fmt.Sprintf("Could not set project %s, trying with another one. Error: %v", projectSlug, err))
 			projectSlug = ""
 			continue
 		}
 		break
 	}
 	if projectSlug == "" {
-		cmd.ui.Info("Successful login. You can now list your projects using 'nerd project list'.")
+		cmd.ui.Info("Successful login, but it seems that there is a connection problem to your cluster(s). Please try again, and if the problem persists, contact mayday@nerdalize.com.")
```

Comment:
@advanderveer do you think this feedback is good enough? Isn't it a bit weird to say "successful login" and at the same time "there is a problem"?